t/Message.t

```diff
@@ -52,6 +52,12 @@ make_path($home_dir);
 dircopy('t/data/list_data/', $home_dir);
 make_path($etc_dir);
 
+## Redirecting standard error to tmp file to prevent having logs all over the output.
+open my $fh, '>', "$tmp_dir/error_log" or die "Can't open file $tmp_dir/error_log in write mode";
+close(STDERR);
+my $out;
+open(STDERR, ">>", \$out) or do { print $fh, "failed to open STDERR ($!)\n"; die };
+
```

Comment:
Really? Because I found loads of logs in the Travis output.
Anyway, it is annoying when running the single test with prove.
Is it is not bug-prone, I suggest to keep it, to make testing smoother.

Reply:
I think the outputs to stderr are useful. I don't agree to throw them away into trashbox file.

Reply:
To control log output:
``` perl
my $log = Sympa::Log->instance;
$log->{log_to_stderr} = 'err';
```

Reply:
Cool ! Thanks !